tests/cli.rs

```diff
@@ -317,3 +317,127 @@ fn proxy_multiple_valid_proxies() {
 // TODO: test implicit download filenames
 // For this we have to pretend the output is a tty
 // This intersects with both #41 and #59
+
+#[test]
+fn verify_default_yes() -> Result<(), Box<dyn std::error::Error>> {
+    let mut cmd = get_command();
+    cmd.arg("-v")
+        .arg("--pretty=format")
+        .arg("get")
+        .arg("https://self-signed.badssl.com");
+
+    cmd.assert()
+        .stdout(predicates::str::contains("GET / HTTP/1.1"));
+
+    cmd.assert().stderr(indoc! {r#"
+    Error: error sending request for url (https://self-signed.badssl.com/): error trying to connect: invalid certificate: UnknownIssuer
+
+    Caused by:
+        0: error trying to connect: invalid certificate: UnknownIssuer
+        1: invalid certificate: UnknownIssuer
+    "#});
```

Comment:
Every time you call `.assert()` it runs the command again, so it's bests to put it all in a single chain.

`.failure()` makes extra sure the command failed. (There's also `.success()`.)

Checking for the precise phrasing and formatting is probably brittle.

So I'd do:
```suggestion
    get_command()
        .arg("-v")
        .arg("--pretty=format")
        .arg("get")
        .arg("https://self-signed.badssl.com");
        .assert()
        .failure()
        .stdout(predicates::str::contains("GET / HTTP/1.1"));
        .stderr(predicates::str::contains("UnknownIssuer));
```
(and so on for the other tests)

Reply:
Can you also change the other tests?

Reply:
Woops, sorry.